### PR TITLE
Update dockerfile and job spec in example

### PR DESCRIPTION
**Pull request by borontion**

### CODE REVIEW

File: lattice/pytorch-lightning-training/job.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    Worker:
+      replicas: 3
+      template:
+        spec:
+          containers:
+            - name: pytorch # must be pytorch
+              image: breezeml/example-pytorch-lightning-training:latest
+              env:
+                - name: "LATTICE_CHECKPOINT_ENDPOINT"
+                  value: "s3://"
```
Comment (johnnt849) [inline lattice/pytorch-lightning-training/job.yaml:16]: I'm a little confused about this. Shouldn't this be something like 's3://bucket-name/path/to/checkpoint/'?

Comment (borontion) [reply lattice/pytorch-lightning-training/job.yaml]: Oh I mean user should write this by himself

Comment (borontion) [reply lattice/pytorch-lightning-training/job.yaml]: I clarified this in the most recent commit.

Comment (johnnt849) [reply lattice/pytorch-lightning-training/job.yaml]: Got it

**Review by johnnt849 — APPROVED** [review]

**Pull request merged**